# This message is for detection results in a generic format
Header header
rasberry_perception/BoundingBox[] bounding_boxes
string[] class_labels